asm/code_80521D0.s

```diff
@@ -4800,7 +4800,7 @@ _08054E0C:
 	adds r1, r5, 0
 	mov r2, r8
 	mov r3, r9
-	bl TrapperOrbAction
+	bl SpikesMoveAction 
```

Comment:
There are trailing spaces in a couple of places within the ASM. Perhaps a bad find/replace?

Reply:
Hmm probably. Will check the rest of these